### ISSUE TITLE
[A11y]: BMT 2 - Document Status - A11y Bug Bash

### ISSUE THREAD

**Issue by bacitracin** [department-of-veterans-affairs/va.gov-team#124817]
# 🔧 Accessibility Support Request

## I am requesting...

**Design / Research**

- [ ] Design review
- [ ] A11y annotations
- [ ] Prototype testing
- [ ] User research planning
- [ ] Tech support for research sessions

**Dev / QA / Training**

- [x] A11y auditing & testing
- [ ] Collaboration Cycle accessibility testing artifact
- [ ] PR Review - *No need to fill this ticket out, just tag me as a reviewer*
- [ ] A11y training
- [ ] Office Hours / Deep Dive

## Background
- **Team name:** BMT 2
- **Product name:** Claims Status Tool
- **Feature name:** Document Status
- **Contact person:** N/A
- **Next Collaboration Cycle touchpoint & date if applicable:** N/A
- **Link to PR/Figma/prototype etc:**  [User Story testing Canvas](https://dsva.slack.com/docs/T03FECE8V/F09QZ0XQ4VB)
- **Links to any related GitHub issues:**
- **Additional details on your request:** Run through the user story tests with Assistive Technology, create new issues for any a11y bugs found

## Yes, I have

- [x] Formatted the GitHub issue title to include: [A11y]: Product - Feature - Request
- [x] Set my project under "Projects"
- [x] Added the appropriate team/project label under "Labels" (e.g., Decision Reviews, BMT1, or BMT2)
- [x] Verified that @bacitracin is assigned to this ticket
